osquery/core/tests/time_tests.cpp

```diff
@@ -0,0 +1,50 @@
+/*
+ *  Copyright (c) 2014-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the BSD-style license found in the
+ *  LICENSE file in the root directory of this source tree. An additional grant
+ *  of patent rights can be found in the PATENTS file in the same directory.
+ *
+ */
+
+#include <gtest/gtest.h>
+
+#include <osquery/core.h>
+#include <osquery/system.h>
+
+#include "osquery/core/utils.h"
+#include "osquery/tests/test_util.h"
+
+namespace osquery {
+
+#ifdef WIN32
+static struct tm* localtime_r(const time_t* timep, struct tm* result) {
```

Comment:
Isn't this same static struct defined in `./osquery/core/system.cpp`? Is that OK?